ntoskrnl/include/internal/cm.h

```diff
@@ -332,6 +340,10 @@ typedef struct _CM_NOTIFY_BLOCK
     ULONG Filter:29;
     ULONG WatchTree:30;
     ULONG NotifyPending:31;
+
+    CM_NOTIFY_MODE NotifyMode;
```

Comment:
Either the following 3 members aren't actually needed
(they are absent from the official symbols, see e.g.:
https://www.vergiliusproject.com/kernels/x86/server-2003/sp2/_CM_NOTIFY_BLOCK
https://www.vergiliusproject.com/kernels/x64/windows-10/22h2/_CM_NOTIFY_BLOCK
https://www.vergiliusproject.com/kernels/x64/windows-11/24h2/_CM_NOTIFY_BLOCK
),
or this means that the notify_block structure is actually being wrap into yet another internal structure, that contains those extra members.

Reply:
I think it might be actually wrapped inside another structure. It was referred as "Post Block" instead of "Notify Block".

Reply:
I just found that "Post Block" is a new internal structure completely different from "Notify Block" and seems like `CM_POST_BLOCK` and `CmpAllocatePostBlock` are not documented anywhere.